resources/views/quizzes/show.blade.php

```diff
@@ -0,0 +1,33 @@
+<x-app-layout page-width="wide">
+    <x-slot name="header">
+    </x-slot>
+    <div class="stack ml-2 mr-2">
+        <x-slot name="title">{{ $title }}</x-slot>
+    </div>
+    <div class="stack ml-2 mr-2">
+        <form class="stack" action="{{ localized_route('quizzes.store-result', $course) }}" method="POST" novalidate>
+            @csrf
+            @foreach ($questions as $question)
+                <fieldset class="field @error('questions.{{ $question->id }}') field--error @enderror">
+                    @if (session('wrongAnswers'))
+                        @if (in_array($question->id, session('wrongAnswers')))
+                            <x-banner type="error">
+                                {{ __('Please try again.') }}
+                            </x-banner>
+                        @else
+                            <x-banner type="success">
+                                {{ __('Correct answer!') }}
+                            </x-banner>
+                        @endif
+                    @endif
```

Comment:
This doesn't appear to be valid markup. The `<legend>` element needs to be the first child of a `<fieldset>`. I suppose you could move the banners above the `<fieldset>` or below `<legend>` depending on where you want them to show up.

see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend